roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
I don't want to lsoe our pre and post rotate settings, and some of the other settings there. Can you do it in a way we retain those other settings and just modify the frequency bits?